### ISSUE TITLE
DEEBOT OZMO T8 AIVI einige Datenpunkte werden nicht befüllt

### ISSUE THREAD

**Issue by senger1985** [mrbungle64/ioBroker.ecovacs-deebot#77]
ecovacs-deebot.0.consumable.filter = Nur %, keine Werte
ecovacs-deebot.0.consumable.side_brush = Nur %, keine Werte
ecovacs-deebot.0.info.battery  = Nur %, keine Werte

ecovacs-deebot.0.history.dateOfLastStartCharging = keine Werte
ecovacs-deebot.0.history.dateOfLastStartCleaning = keine Werte
ecovacs-deebot.0.history.timestampOfLastStartCharging = keine Werte
ecovacs-deebot.0.history.timestampOfLastStartCleaning = keine Werte
ecovacs-deebot.0.info.chargestatus = keine Werte
ecovacs-deebot.0.info.cleanstatus = keine Werte
ecovacs-deebot.0.info.debugMessage = keine Werte

Und es ist wird keine Map erstellt.

**Comment by mrbungle64** [comment]
Hallo @senger1985 

Der OZMO T8 AIVI ist ja noch neu und noch nicht unterstützt.
Trotzdem schon mal danke für die Infos :+1: 

Damit wir die Unterstützung des Modells hinbekommen erst einmal ein paar Fragen:

* was funktioniert denn bereits schon?
* was wird im Log nach einem Start des Adapters in der Zeile die mit "Device[0]" beginnt als Wert zu "company" ausgegeben?
* Unter "info.deviceClass" steht der Wert "x5d34r". Ist das korrekt?
* Ist es möglich, dass Du auf dem System (Raspberry Pi? Linux?) mal auf der Konsole was ausführst um uns ein paar Log-Auszüge zu posten?

> Und es ist wird keine Map erstellt.

Was meinst Du damit?

**Comment by senger1985** [comment]
Zu 1: Die Controls habe ich noch nicht getestet(wird demnächst gemacht)

Zu 2: (26973) Device[0]: {"did":"04230e27-52ce-42d2-9fb6-0776311a9fdd","name":"E03Q14004B09HCKJ0279","class":"x5d34r","resource":"ehmi","nick":"Robby 2.0","company":"eco-ng","bindTs":1591707739853} (Bitte sofort bescheid geben, sollte da irgendwelche sensiblen Daten sein, ich ich nicht gefiltert habe)

Zu 3: Ja

Zu 4: Kann ich heute, zum späten Abend gerne machen

Zu 5: Ich dachte, man kann irgendwo die Map, des Deebots, anzeigen lassen, wo er gereinigt hat.

**Comment by mrbungle64** [comment]
> Zu 1: Die Controls habe ich noch nicht getestet(wird demnächst gemacht)

ok, aber der Bot hat mal einen Reinigung durchgeführt während der Adapter bereits gestartet war?

> 
> Zu 2: (26973) Device[0]: {"did":"xxx","name":"xxx","class":"x5d34r","resource":"xxx","nick":"xxx","company":"eco-ng","bindTs":1591707739853} (Bitte sofort bescheid geben, sollte da irgendwelche sensiblen Daten sein, ich ich nicht gefiltert habe)

Danke, habe alles direkt in Deinem Post durch xxx ersetzt, was ich nicht benötige.

> 
> Zu 3: Ja
> 
> Zu 4: Kann ich heute, zum späten Abend gerne machen

Alles klar. Ich poste das dann später mal, was Du dann machen könntest.

> 
> Zu 5: Ich dachte, man kann irgendwo die Map, des Deebots, anzeigen lassen, wo er gereinigt hat.

Eine Live-Map gibt es (noch) nicht. Es sollte nach dem Reingen unter "cleaninglog.lastCleaningMapImageURL" einen URL drin sein, unter welcher man die letzte Reinigungskarte abrufen kann. Aber das ist noch experimentell und muss auch extra angeschaltet werden. Außerdem hat Ecovacs selbst mit den Reinigungsprotokollen (inkl. der Map) noch Probleme.

**Comment by senger1985** [comment]
Ja, der Adapter lief, als der Bot eine Reinigung durchgeführt hat

Den Datenpunkt "cleaninglog.lastCleaningMapImageURL" wird bei mir gar nicht angezeit.
Bzw. der ganze Datenbau "cleaninglog"
Aber okay, wenn das noch irgendwie angeschalten werden muss, kann es bei mir ja auch nicht angezeigt werden.

**Comment by mrbungle64** [comment]
@senger1985 

Ich würde dann erst mal mit den Akku- und Verbrauchsmaterialwerten anfangen wollen.
Es wäre cool, wenn Du mal folgendes auf dem ioBroker-System machen würdest:

1)
```bash
git clone https://github.com/mrbungle64/ecovacs-deebot.js.git
cd ecovacs-deebot.js
npm install
cd example
```

2) Email und Passwort in der Datei `app2.js` anpassen

3) Die app ausführen 
```bash
NODE_ENV=dev node app2.js | grep -E 'Battery|LifeSpan'
```

4) laufen lassen bis ein paar Werte auftauchen (ca. 30 Sek. oder so) und anschließend die Ausgabe posten

Unabhängig davon:
Hast Du schon mal geschaut, ob im ioBroker Log etwas auffälliges zu sehen ist?

**Comment by senger1985** [comment]
Ich habe dir eben eine Mail geschickt

**Comment by senger1985** [comment]
Controls werden morgen getestet
